napari/components/_interaction_box_mouse_bindings.py

```diff
@@ -183,20 +183,23 @@ def _on_mode_change(self, event):
     def _on_transform_change(self, event):
         """Gets called when the interaction box is transformed to update transform of the layer"""
 
+        viewer = self._ref_viewer()
+        active_layer = viewer.layers.selection.active
+        if active_layer is None:
+            return
+
         layer_dims_displayed = dims_displayed_world_to_layer(
             list(self._ref_viewer().dims.displayed),
-            self._ref_viewer().dims.ndim,
-            self._ref_viewer().layers.selection.active.ndim,
+            viewer.dims.ndim,
+            active_layer.ndim,
         )
 
         layer_affine_transform = event.value.compose(
             self._initial_transform_inverse
         )
```

Comment:
Instead of changing `Transform.compose` to handle `None` (which is an API change), we could just special case things here as a simpler quick fix.

```suggestion
        layer_affine_transform = event.value
        if self._initial_transform_inverse is not None:
            layer_affine_transform = layer_affine_transform.compose(self._initial_transform_inverse)
```

Reply:
oh. I miss `layer_affine_transform = event.value` line and implement my own version.